python-ecosys/aiohttp/aiohttp/__init__.py

```diff
@@ -18,8 +18,14 @@ class ClientResponse:
     def __init__(self, reader):
         self.content = reader
 
+    def get_header(self, keyname, value=None):
+        try:
+            return next((v for k, v in self.headers.items() if k.lower() == keyname.lower()))
+        except StopIteration:
+            return value
```

Comment:
To make this more efficient for MicroPython I suggest:
```py
for k in self.headers:
    if k.lower() == keyname:
        return self.headers[k]
return default
```
Then make sure `keyname` is always lowercase when passed in.